src/main/java/com/denizenscript/depenizen/bukkit/objects/luckperms/LuckPermsTrackTag.java

```diff
@@ -130,43 +142,86 @@ public ObjectTag getObjectAttribute(Attribute attribute) {
         // @description
         // Returns the name of the track.
         // -->
-        if (attribute.startsWith("name")) {
-            return new ElementTag(track.getName()).getObjectAttribute(attribute.fulfill(1));
-        }
+        tagProcessor.registerTag(ElementTag.class, "name", (attribute, object) -> {
```

Comment:
This could maybe be static as well, assuming it's similar to the case with groups? should test first to make sure ofc